src/components/Banners/Banner.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { PureComponent } from 'react'
+import cx from 'classnames'
+
+import Button from 'cozy-ui/react/Button'
+import { SubTitle } from 'cozy-ui/react/Text'
+
+export class Banner extends PureComponent {
+  render() {
+    const {
+      buttonLabel,
+      buttonIcon,
+      description,
+      disabled,
+      isImportant,
+      onButtonClick,
+      title
+    } = this.props
```

Comment:
propTypes

Reply:
I'm not a big fan of adding several props for the same "component". I prefer this kind of solution  (but it's not a blocking comment for now ;)) 

```jsx 
<Banner 
title="title"
description="desc"
button=<Button onClick={fn} label={label} icon={icon} />
/>
```